tezos/encoding-derive/src/common.rs

```diff
@@ -0,0 +1 @@
+pub type Result<T> = std::result::Result<T, syn::Error>;
```

Comment:
@akoptelov
do we really need this `common.rs` ? why not to put this to `lib.rs`

Reply:
It is moved to `make.rs`